mackerel/resource_mackerel_role_test.go

```diff
@@ -29,7 +29,8 @@ func TestAccMackerelRole(t *testing.T) {
 					testAccCheckMackerelRoleExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "service", serviceName),
 					resource.TestCheckResourceAttr(resourceName, "name", name),
-					resource.TestCheckResourceAttr(resourceName, "memo", ""),
+					// TODO: For backword compatibility, we don't test "memo".
+					// resource.TestCheckNoResourceAttr(resourceName, "memo"),
```

Comment:
This statement has tested that the provider produces inconsistent state after apply (`memo` is null in the plan).
It seems to be possible to support this behavior with a [plan modification](https://developer.hashicorp.com/terraform/plugin/framework/resources/plan-modification), but it is too much.